### PR TITLE
Add afl-2.57b package.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
This is the last official release from https://github.com/google/AFL/releases.

It's not clear how I can test this locally within opam. I have applied the patch manually to the project source and that works ok. Submitting here to get testing on various platforms.

### CODE REVIEW

**Comment by avsm** [comment]
Just restarting a failed macos/arm build (SIGBUS on git reset?!), but otherwise looks fine.

File: packages/afl/afl.2.57b/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+remove: [
+  [make "PREFIX=%{prefix}%" "uninstall"]
+]
+patches: [
+  "add-uninstall-target.patch"
+]
+synopsis:
+    "American Fuzzy Lop fuzzer by Michal Zalewski, repackaged for convenient use in opam"
+authors: "Michal Zalewski"
+depends: ["ocaml"]
```
Comment (mseri) [inline packages/afl/afl.2.57b/opam:24]: I don't understand this, the package only depends on a c compiler, no?

Comment (avsm) [reply packages/afl/afl.2.57b/opam:24]: Woops, I merged before seeing this. I think @mseri is right...

Comment (tmcgilchrist) [reply packages/afl/afl.2.57b/opam:24]: Yes it only depends on a c compiler. 
What's the correct way to depend on that? @mseri

Comment (mseri) [reply packages/afl/afl.2.57b/opam:24]: I think it is `(conf-gcc | conf-clang)` but I am not 100% sure. If there is ocaml, most likely there is also the c compiler, so even though it is not strictly required perhaps is a good idea to keep it as is

**Pull request merged**